lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -3528,6 +3528,14 @@ static int prLanguageConfig_removeExcludePath(struct VMGlobals * g, int numArgsP
 	return prLanguageConfig_removeLibraryPath(g, numArgsPushed, excludePaths);
 }
 
+static int prLanguageConfig_getCurrentConfigPath(struct VMGlobals * g, int numArgsPushed)
+{
+	PyrSlot *a = g->sp;
+	PyrString* string = newPyrString(g->gc, gLanguageConfig->getCurrentConfigPath(), 0, false);
```

Comment:
Ah I assumed it would always exist, which is obviously wrong. You know how to fix?